hms-app/src/components/card-details/HackathonDetails.tsx

```diff
@@ -50,9 +50,15 @@ const useStyles = createStyles((theme) => ({
   },
 }))
 
+enum Enum {
```

Comment:
should also have a bit better name than just `Enum`, `HackathonDetailsType` maybe